components/uploadForm/final.tsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import { EStep } from '../../constants';
+import SuccessIcon from '../../static/success.png';
+import FailureIcon from '../../static/failure.png';
+import WarningIcon from '../../static/warning.png';
+
+interface IProp {
+  step: EStep;
+  setStep: React.Dispatch<React.SetStateAction<EStep>>;
+  data?: any;
+}
+
+const Final: React.FC<IProp> = (props: IProp) => {
+  switch (props.step) {
```

Comment:
If UI is the same component, seems we don't need to use `switch case` ?
We can refactor by determining the variable on the component top level, and pass it down.

like this:
```javascript
const color = ... // 依據 step 去判斷要給哪個顏色
const title = ... // 依據 step 去判斷要給哪個標題
const description = ... // 依據 step 去判斷要給哪個描述
const buttonText = ... 

return (
<>
  <Title style={{ ..., color, ... }}>{title}</Title>
  <Description>{description}</Description>
  ...
  ...
</>
)


```